### ISSUE TITLE
Fingerprint capturing/enrolment solution?

### ISSUE THREAD

**Issue by leonvr** [cobisja/tad-php#65]
Hi
This may be a long-shot....but...
Has anybody found a PHP-based fingerprint capture/enrolment system to capture the fingerprints to database?

**Comment by whoamikenken** [comment]
Yeah but image captured template is only base64 the attendance has an algorithm of vx9.0. I'm also looking for a way to get their algorithm.

**Comment by Sarojb31** [comment]
how did you capture the image and vx 9 template .... please help me in this?

**Comment by whoamikenken** [comment]
Use the get_user_template function look into it the function... I already developed it but you still need the device to register unto
